typescript/sdk/src/providers/transactions/transformer/ethersV5/EV5InterchainAccountTxTransformer.ts

```diff
@@ -56,19 +65,23 @@ export class EV5InterchainAccountTxTransformer
       this.props.config,
     );
 
-    const transformedTxs: ethers.PopulatedTransaction[] = [];
-    for (const txChain of objKeys(txChainsToInnerCalls)) {
+    const transformedTxs: Promise<ethers.PopulatedTransaction>[] = [];
+    objMap(txChainsToInnerCalls, (destination, innerCalls) => {
       transformedTxs.push(
-        await interchainAccountApp.getCallRemote({
+        interchainAccountApp.getCallRemote({
           chain: this.props.chain,
-          destination: txChain,
-          innerCalls: txChainsToInnerCalls[txChain],
+          destination,
+          innerCalls,
           config: this.props.config,
           hookMetadata: this.props.hookMetadata,
         }),
       );
-    }
+    });
 
-    return transformedTxs;
+    return await concurrentMap(
```

Comment:
nit: unnecessary await
ive mentioned this a few times now

Reply:
^noting that this has now been done in another commit